tests/test_commands.py

```diff
@@ -169,6 +180,12 @@ def test_drop_existing_db(mock_rethink_db_drop):
     run_drop(args)
 
 
+def test_drop_non_existing_db(mock_rethink_db_drop_not_exist):
+    from bigchaindb.commands.bigchain import run_drop
+    args = Namespace(config=None, yes=True)
+    run_drop(args)
```

Comment:
> One approach would be to propagate the underlying rethink db status through run_drop (perhaps translating it to some bigchaindb format) and then to the testing code. This could then easily be asserted (looking at the returned tables_dropped field, for example).

Those `run_*` functions are just used as a bridge from the command line to the actual code to run, so they prepare/set up the state for BigchainDB to work.

> What kind of improvements did you have in mind?

In the test you can check if `stderr` contains the error message you expect. You can use `capsys`, check it out here:
 - https://github.com/bigchaindb/bigchaindb/blob/master/tests/test_commands.py#L111